app.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const app = express();
+const {
+    getUsers,
+    deleteUser,
+    editUser,
+    updateUser,
+    createUser,
+    paginateUsers
+} = require("./userService");
+
+app.use(express.static('public'));
+
+const port = process.env.PORT
+
+app.get("/", function(req, res) {
+    res.render("index.pug");
+});
+
+// Get Users
+app.get("/users", function(req, res) {
+    getUsers(req.query, res);
+});
+
+// Create a User
+app.post("/users", function(req, res) {
+    res.sendStatus(201);
```

Comment:
you send response before user is created. The client knows nothing regardless of was the user created or not

Reply:
Fixed. (I think)